### DISCUSSION TITLE
DNN SMTP Test Settings

### DISCUSSION THREAD

**Discussion by thabaum** [dnnsoftware/Dnn.Platform#4088] [General] [answered]
From this point in my history there was two contributors to the file:

![image](https://user-images.githubusercontent.com/13577556/93261145-2ef14280-f757-11ea-9bda-ef19b8c5663b.png)

```              var smtpHostMode = request.SmtpServerMode == "h";
                var mailFrom = Host.HostEmail;
                var mailTo = smtpHostMode ? Host.HostEmail : PortalSettings.UserInfo.Email;
```

Then I contributed this patch:

```                
var mailFrom = smtpHostMode ? Host.HostEmail : PortalSettings.Email;              
var mailTo = UserInfo.Email;
```

https://github.com/dnnsoftware/Dnn.Platform/pull/3237/commits/3870357fda51370135ad7944bb20a78850c86374

But recent version shows this 

![image](https://user-images.githubusercontent.com/13577556/93261284-5a742d00-f757-11ea-849e-17377ccea3a9.png)

Still contains the logic I contributed however `this.` has been added.

```
                var mailFrom = smtpHostMode ? Host.HostEmail : this.PortalSettings.Email;
                var mailTo = this.UserInfo.Email;
```

I am trying to chase some issues in the smtp not using the portal admin email on a portal but seems to work fine on other portals I believe... I was trying to go back through the code and changes... it seems my name has been removed from this file as a contributor, so I am asking how and why?  Was there a regression?  It looks like same code logic I contributed just no credit, I was wondering how I disappeared from the file.

https://github.com/dnnsoftware/Dnn.Platform/pull/3237

**Comment by bdukes** [comment, marked as answer]
I think it probably got lost in a merge.  Since it's just a one-line change, when someone else touched that line (to adjust styles and add `this.`), you were no longer an "owner" of that line change.  It is weird that GitHub still doesn't show you as a contributor, but I guess it figures that figuring out contributors line-by-line is usually good enough (and prevents old contributors from showing up, for good or bad).

**Reply by thabaum** [reply]
OK, I was just curious I figured that the this addition would probably have done it.
